src/audio/Kconfig

```diff
@@ -338,6 +338,16 @@ config COMP_CODEC_ADAPTER
 	  "src\include\sof\audio\codec_adapter\interfaces.h". It is possible to link several
 	  different codecs and use them in parallel.
 
+config COMP_IGO_NR
+	depends on XCHAL_HAVE_HIFI3
+	bool "IGO NR component"
+	default n
```

Comment:
add depends on `FORMAT_S16LE` since you are locked to a particular format

Reply:
Actually we accept S16_LE and S32_LE in the wrapper. In the case of S32_LE we just right shift the data by 16 bits.

I will rewrite the code to make it more clear.